src/css/jwplayer/flags/floatingplayer.less

```diff
@@ -10,6 +10,13 @@
         height: auto;
         margin-top: 2em;
         position: fixed;
+
+        &.jw-flag-small-player {
```

Comment:
Couldn't we just make this a media query to match the AC "Small screen portrait"?